torch/_dynamo/decorators.py

```diff
@@ -2,9 +2,12 @@
 # ruff: noqa: TCH004
 import functools
 import inspect
+import sys
 from dataclasses import dataclass
 from typing import Any, Callable, Dict, Type, TYPE_CHECKING, TypeVar
 
+from packaging import version
```

Comment:
We should never used `packaging`, but instead use `torch_version` which tries to use packaging, and if not available has some sort of home-grown implementation

Reply:
Sounds good, thanks!!